css/templates/themes/2/page.css

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2017 The AMPStart Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../core';
+@import '../../../../components/footer/footer';
+@import '../../../../components/images/fullpage-hero';
+@import '../../../../components/images/images';
+@import '../../../../components/navbar/navbar';
+@import '../../../../components/social-follow/social-follow';
+@import 'page-vars';
+
+body {
+  line-height: 1.75;
+}
+
```

Comment:
I’ve tried this locally and it’s caused quite a few problems, because of the way these variables are being (ab)used in other places, such as the `--navbar-height` variable and the `line-height` in the `.ampstart-navbar-trigger` ruleset.

Just because an element has the same height as another one doesn’t necessarily mean they should be using the same variable, unless there’s a design requirement for the dimensions to be explicitly dependent on one another.

In this case, I don’t think the `--navbar-height` variable and `.ampstart-navbar-trigger` styling should be dependent on either the `--line-height-6` or `--h2` variables, since those should be used exclusively for header styling.

How do you this should be resolved?